### PR TITLE
Ui list

### PR DESCRIPTION
**Pull request by kwood15**
Please review: Story - #23

Screenshots here:

screen shot 2017-09-15 at 13 55 11

screen shot 2017-09-15 at 13 55 25

### CODE REVIEW

File: nebula-components/tools/_ui-list.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@mixin nb-ui-list {
```
Comment (rbrtsmith) [inline nebula-components/tools/_ui-list.scss]: Not much point in making this a mixin.

File: nebula-components/objects/_ui-list.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+@if $nb-use-ui-list {
+  .#{$nb-namespace}o-ui-list {
+    @include nb-ui-list;
```
Comment (rbrtsmith) [inline nebula-components/objects/_ui-list.scss]: This should just be `@include nb-bare-list`

File: nebula-components/objects/_ui-list.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  }
+
+  .#{$nb-namespace}o-ui-list__item--divider {
```
Comment (rbrtsmith) [inline nebula-components/objects/_ui-list.scss]: Just like the `bare-list` I'd go with `o-ui-list__item` as the classname as this isn't just a divider, it contains content also.

File: nebula-components/objects/_ui-list.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    border-bottom: 1px solid $nb-color-base-border;
+  }
+
+  .#{$nb-namespace}o-ui-list__item--divider:last-child {
```
Comment (rbrtsmith) [inline nebula-components/objects/_ui-list.scss]: If we do
```
.#{$nb-namespace}o-ui-list__item + .#{$nb-namespace}o-ui-list__item {
  border-top: 1px solid $nb-color-base-border;
}
```
We won't need to remove the border on the last child.  I'd argue this is preferable as we avoid setting a style prematurely and then undoing it.

**Comment by rbrtsmith** [comment]
Also need to add spacing to this.  can use the mixin like here: https://github.com/rbrtsmith/nebula-css/blob/master/nebula-css/objects/_bare-list.scss#L5.
`@include nb-list-spacing('ui-list', 'bottom');`
the generated classname from that should then be `.#{$nb-namespace}o-ui-list--spaced-md`

File: nebula-components/tools/_ui-list.scss (hunk 1 above)
Comment (kwood15) [reply nebula-components/tools/_ui-list.scss]: done

File: nebula-components/objects/_ui-list.scss (hunk 2 above)
Comment (kwood15) [reply nebula-components/objects/_ui-list.scss]: done

File: nebula-components/objects/_ui-list.scss (hunk 3 above)
Comment (kwood15) [reply nebula-components/objects/_ui-list.scss]: done

File: nebula-components/objects/_ui-list.scss (hunk 4 above)
Comment (kwood15) [reply nebula-components/objects/_ui-list.scss]: done

**Comment by kwood15** [comment]
Fixes done as above:
Here is the spacing with the default md:
screen shot 2017-09-15 at 15 32 42

**Comment by rbrtsmith** [comment]
Closes #23

**Review by rbrtsmith — APPROVED** [review]

**Pull request merged**